targets/unity-v2/Testing/Tests/Entity/EntityApiTests.cs

```diff
@@ -319,10 +315,10 @@ void TestFileContent(UUnitTestContext testContext, byte[] result)
         {
             var testFileData = Encoding.UTF8.GetString(result);
 
-            testContext.True(testFileData.Equals(_testPayload));
+            testContext.True(testFileData.Equals(_testPayload), testFileData);
```

Comment:
Alex, can you help me figure out why this test is failing here?
testFileData is null.

Reply:
Only happens after my changes, but I don't know why.